### ISSUE TITLE
Improve Get FoCo Accessibility 

### ISSUE THREAD

**Issue by penoptech** [Get-Your/GetFoco#156]
This will likely fall in with the UI redesign, but I wanted to make sure it gets logged and we don't lose sight of improving Get FoCo's accessibility. Below is a short checklist for what we need to ensure as apart of improve Get FoCo's accessiblity:

- Images should have meaningful alternative text
- Links should be visually identifiable
- Use descriptive section headings
- Use correct semantic HTML element structure for your content
- Forms have descriptive labels
- Text and background color should have sufficient contrast
- Content scales properly when zoomed/enlarged
- Use a descriptive title tag
- Support keyboard navigation
- Focus states should be visible for keyboard users
- Use correct HTML5 input types
- Content that automatically changes has the ability to be paused
- Limit or remove any flashing elements
- Users should be able to navigate content using a screen reader
- Allow keyboard users to skip navigation
- Offer multiple ways to find pages on your website
- Avoid mouse only interactions
- Set focus on modals, popovers, alerts etc.
- Site should not time out unexpectedly
- Multimedia should have alternative ways to be consumed
- Use the HTML lang attribute
- Use understandable inputs labels
- Links should be descriptive and provide intent
- Table data is accessible to non-sighted users
- Use ARIA landmarks where applicable
- Decorative images should not be visible to screen readers
- Pages are understandable with no styles enabled
- Web page size should not exceed 500k
- HTML should be valid and error-free
